bundles/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -114,3 +132,29 @@ then
 end 
 
 ```
+
+dwdunwetter_severity.map
+
+```
+Minor=Wetterwarnung
+Moderate=Markante Wetterwarnung
+Severe=Unwetterwarnung
+Extreme=Extreme Unwetterwarnung
+NULL=undefiniert
+```
+
+
+
+dwdunwetter_urgency.map
```

Comment:
Please rename these transformation files to `dwdunwetter_urgency_de.map` as it contains German mappings.

Reply:
Thanks... Changed and fixed